app/controllers/service_controller.rb

```diff
@@ -200,6 +192,16 @@ def service_form_fields
 
   private
 
+  def features
+    [{:role     => "service",
+      :role_any => true,
+      :name     => :svcs,
+      :title    => N_("Services")},
+    ].map do |hsh|
+      ApplicationController::Feature.new_with_hash(hsh)
+    end
+  end
+
```

Comment:
Here too, we have just one element to map over. As you already fixed earlier in other place.